driver/cassandra/cassandra.go

```diff
@@ -83,16 +58,16 @@ func (driver *Driver) Close() error {
 }
 
 func (driver *Driver) ensureVersionTableExists() error {
-	err := driver.session.Query("CREATE TABLE IF NOT EXISTS " + tableName + " (version counter, versionRow bigint primary key);").Exec()
-	if err != nil {
-		return err
-	}
 
-	_, err = driver.Version()
-	if err != nil {
-		driver.session.Query(up.String(), versionRow).Exec()
+	if err := driver.session.Query("CREATE TABLE IF NOT EXISTS " + tableName +
+		" (driver_version int," +
+		"version bigint," +
+		"file_name text," +
```

Comment:
about the `driver_version` it was the tricky part of this.
For structural reason cassandra can do `ORDER BY` only of the second part of a primary key.
So I have to have a first part...